localstack/aws/protocol/service_router.py

```diff
@@ -270,12 +272,36 @@ def get_service_catalog() -> ServiceCatalog:
         return ServiceCatalog()
 
 
-def resolve_conflicts(candidates: Set[str], request: Request):
+_IVS_REALTIME_STAGE_RE = re.compile(r"^arn:aws:ivs:[a-z0-9-]+:[0-9]+:stage/[a-zA-Z0-9-]+$")
```

Comment:
Wow, the changes in here are really elaborate. And these will be quite helpful once we decide to support `ivs` or `ivs-realtime`. But at this point in time I would suggest to actually exclude these services from the list of tested services in the `test_service_router` (see comment below), because:
- The service router is a very fundamental heuristic in LocalStack. It is already pretty complex, and we want to avoid adding complexity which isn't absolutely necessary.
- The service router is executed for each and every request that is sent to LocalStack. This is why we want to keep it as slim and small as possible.
- We currently don't support `ivs` or `ivs-realtime`, which is why - even if we can detect the requests to these services correctly - we can't do anything with it anyways yet.

Reply:
cool, that's helpful, I didn't know if ivs was supported (suspected not), and suspected there might be a simpler way to deal with this.

Reply:
removed and excluded ivs